packages/spor-react/src/datepicker/DatePicker.tsx

```diff
@@ -76,33 +85,33 @@ export const DatePicker = forwardRef<HTMLDivElement, DatePickerProps>(
       ref as React.MutableRefObject<HTMLDivElement>
     );
 
+    const styles = useMultiStyleConfig("Datepicker", {});
+    const locale = useCurrentLocale();
+
     const responsiveVariant =
       useBreakpointValue(typeof variant === "string" ? [variant] : variant) ??
       "simple";
-
-    const locale = useCurrentLocale();
-
-    const handleEnterClick = (e: React.KeyboardEvent) => {
-      if (
-        responsiveVariant === "simple" &&
-        e.key === "Enter" &&
-        !state.isOpen
-      ) {
-        // Don't submit the form
-        e.stopPropagation();
-        state.setOpen(true);
```

Comment:
I struggled finding that this code is really doing anything, other than creating confusion... I tried using Enter within the input field, but the calendar does not open...

Reply:
Hm that's unfortunate. Without this code, there isn't really any way a screen reader can trigger the calendar. 

Not much of a difference since it didn't work anyways, but we should probably find a way to trigger the calendar for the simple variant, right?

Reply:
The normal way to open stuff when navigating with a screen reader is in fact by using space, not enter - and that works in the simple variant when I focus at the label. But keyboard users will not be able to focus on this element.. I do not know what has been discussed about the simple variant, but maybe rather it should open automatically as there is no actual indication that the user should press either space or enter in the input field to open it. 

But I will leave it as is until now and this can be discussed separately 👍